app/javascript/app/components/nav/nav-component.jsx

```diff
@@ -43,7 +43,11 @@ class Nav extends PureComponent {
               exact={route.exact || false}
               className={cx(styles.link, theme.link)}
               key={route.slug}
-              to={route.link || route.path}
+              to={
+                route.link
+                  ? `/${locale}${route.link}`
+                  : `/${locale}${route.path}`
```

Comment:
I think we can simplify it to `to={`/${locale}${route.link || route.path}`}`, but it's just the suggestion :)

Reply:
well spotted, thanks